test/README.md

```diff
@@ -285,7 +285,7 @@ Number; !must be defined to function properly):
 
 - `VALGRIND_LOG` (F) (S): overrides valgrind log file name used for `VALGRIND`.
 
-- `TEST_COLORS` (F) (U) (D): enable pretty colors in test runner.
+- `NO_TEST_COLORS` (F) (U) (D): disable pretty colors in test runner.
```

Comment:
avoid negative-logic. no need to rename this, just document that the default is "true".